cmd/sqlcmd/sqlcmd.go

```diff
@@ -126,9 +126,11 @@ func Execute(version string) {
 		PreRunE: func(cmd *cobra.Command, argss []string) error {
 			SetScreenWidthFlag(&args, cmd)
 			if err := args.Validate(); err != nil {
```

Comment:
I've tested this PR and it fixes the issue.  But while testing it, I see that we have a new issue post v1.0 with connecting to localhost and/or the current context when running "sqlcmd bad-sub-command", when it should give an error"

Can you fix that issue in this PR as well?